src/rfc3339.rs

```diff
@@ -0,0 +1,61 @@
+use chrono::{format::ParseError, offset::TimeZone, DateTime, FixedOffset, SecondsFormat};
+use core::{
+    cmp::Ordering,
+    fmt::{self, Display, Formatter},
+    str::FromStr,
+};
+
+#[derive(Clone, PartialEq)]
```

Comment:
Is the derive of `PartialEq` needed? If yes, should it be a custom implementation to compare with `self.1`, in the veryyyyy unlikely case that they use different time precision?

Reply:
very good question, not sure if equality for this type means exact same string representation or not (currently yes, but no real thought was put into it)

Reply:
comparison of the timestamps can be done as `a.as_ref() == b.as_ref()` or via the `impl PartialEq<DateTime<T>>`, but it's cumbersome and not at all obvious to the user

Reply:
I was just thinking that a user probably shouldn't need to test equality of timestamps, but it's no big deal.